integ-test/src/test/java/org/opensearch/sql/legacy/OpenSearchSQLRestTestCase.java

```diff
@@ -66,6 +66,9 @@ protected RestClient buildClient(Settings settings, HttpHost[] hosts) throws IOE
   }
 
   protected static void wipeAllOpenSearchIndices() throws IOException {
+    if (client() == null) {
```

Comment:
feels like this check should be moved to `RestIntegTestCase::cleanUpIndices` and `SQLIntegTestCase::cleanUpIndices` (those classes already check for client() on setup)

Reply:
45d97bbf9